### PR TITLE
bump version

### PR DESCRIPTION
**Pull request by yakkomajuri**
## Changes

dont shame me bot

- bump version to 1.22.0
- update changelog

## Checklist

- [ ] All querysets/queries filter by Organization, by Team, and by User
- [ ] Django backend tests
- [ ] Jest frontend tests
- [ ] Cypress end-to-end tests

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,79 @@
...
+
+
+- [Project API Key Autofill in Docs for Cloud Users](https://github.com/PostHog/posthog.com/pull/998)
+
+![Docs Token Autofill](https://posthog-static-files.s3.us-east-2.amazonaws.com/Website-Assets/Array/token-autofill.png)
+
+If you're a user of PostHog Cloud, we now autofill your Project API Key and API Host automatically in the Docs for you, meaning you can copy-paste snippets and use them directly with no manual changes!
+
+This key will be based on the last project you used in PostHog, and you can check what project that is by simply hovering your cursor over the highlighted key.
+### 1.21.0 - Wednesday 17 February 2021
```
Comment (EDsCODE) [inline CHANGELOG.md:77]: #2952 is a pretty major feature that's still behind a feature flag. not sure if these are included in changelog or not

Comment (yakkomajuri) [reply CHANGELOG.md:89]: No, those behind feature flags don't go in - There's quite a few big features that remain unreleased: funnel trends, formulas, cohorts on modal, labels

Comment (yakkomajuri) [reply CHANGELOG.md:89]: Will make for a nice 1.23.0 :smiley:

**Comment by yakkomajuri** [comment]
Heads up for a flaky test

**Pull request merged**